hobbit-sharad.cpp

```diff
@@ -0,0 +1,23 @@
+#include <iostream>
+
+using namespace std;
```

Comment:
std is a namespace. cout is present in the namespace std. I could have either done std::cout or add a <code>using namespace std;</code> and use everything inside std directly.  Read more here: http://www.tutorialspoint.com/cplusplus/cpp_namespaces.htm